seleniumbuilder/content/html/builder/record.js

```diff
@@ -81,25 +81,25 @@ builder.record.stop = function() {
 
 builder.record.continueRecording = function(insertionIndex) {
   jQuery('#record-panel').show();
-  
+
```

Comment:
A lot of "emtpy" lines with spaces.. I'm not sure why is that. My editor automatically removes the spaces from the empty lines. I hope is not a problem.